sunscreen_runtime/src/debugger/server.rs

```diff
@@ -321,3 +358,291 @@ pub async fn get_stack_trace(
     }
     Ok(HttpResponse::NotFound().body(format!("Session {} not found", session)))
 }
+
+#[cfg(feature = "debugger")]
+#[get("sessions/{session}/groups/{groupid}")]
+pub async fn get_group(
+    path_info: web::Path<(String, usize)>,
+) -> Result<HttpResponse, actix_web::Error> {
+    let (session, groupid) = path_info.into_inner();
+    let sessions = get_sessions().lock().unwrap();
+
+    if let Some(curr_session) = sessions.get(&session) {
+        match curr_session {
+            Session::BfvSession(s) => {
+                let lookup = &s.graph.metadata.group_lookup;
+
+                if let Some(curr_group) = lookup.id_data_lookup.get(&groupid.try_into().unwrap()) {
+                    let child_groups = lookup.children_of(groupid.try_into().unwrap());
+
+                    let mut graph = s.graph.filter_map(
```

Comment:
I would move this to a helper function called `generate_fhe_display_graph`. Ditto for the ZKP case below and share as much code as you can.